data/ghosts.json

```diff
@@ -114,7 +114,7 @@
             "evidence": [
                 "Ultraviolet",
                 "Ghost Orbs",
-                "DOTs"
+                "DOTS"
```

Comment:
This one needs to stay with the lower case 's' or it will break the shared journal link and desktop link

Reply:
must have accidentally changed that one, sorry!